src/components/Quote.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+
+const Quotes = () => {
+  const [data, setData] = useState(null);
+  useEffect(() => {
+    const fetchData = async () => {
+      const res = await fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+        method: 'GET',
+        headers: {
+          'X-Api-Key': '9EoCxmipJCj+zR35ZNsY3A==93CN5SnCNPUiktDW',
+          'Content-type': 'application/json',
+        },
+      });
+      const data = await res.json();
+      if (data.length > 0) {
+        setData(data[0].quote);
+      } else {
+        setData('');
+      }
+    };
+    fetchData();
+  }, []);
+  return (
+    <div className="Quotes">
+      {data ? (<p>{data}</p>) : (
+        <p>Loading....</p>
+      )}
+    </div>
+  );
+};
+
+export default Quotes;
```

Comment:
- Please ensure your `Quotes` component has an error state to handle errors according to the project requirement 👍🏽 
![Screenshot from 2023-04-27 16-09-39](https://user-images.githubusercontent.com/44624138/234906774-82f4cd9c-1b0b-4f11-a8d5-ad9897ebdf2c.png)